### ISSUE TITLE
Danq val_acc_g_row metric can be nan

### ISSUE THREAD

**Issue by soi** [weberlab-hhu/GeenuFF#98]
Theoretically no bug per se but only happens predictably at the third epoch, which means the validation set are not the same.. Which is not tested.

Could also look into keras sequences to make this less bug prone.
